build.sbt

```diff
@@ -1169,6 +1181,7 @@ lazy val `sbt-build-tool-support` = (project in file("dev") / "build-tool-suppor
 
 lazy val `sbt-plugin` = (project in file("dev") / "sbt-plugin")
   .settings(common: _*)
+  .settings(noMima)
```

Comment:
I'm also in doubt if we need to check binary compatibility for the plugin. My initial reaction is that, yes, we need to do that. Maybe we can be more relaxed in situations where it does not impact users, but what if we remove/rename a setting, for example?

Reply:
Obvious incompatibilities like that will be obvious in reviews.